lessons/module-2/context-and-this-keyword.md

```diff
@@ -25,32 +25,32 @@ As we have discussed earlier, the keyword *this* in JavaScript can be confusing.
 Consider the following `Unicorn` class:
```

Comment:
It looks like we still have a reference to `class` here.  Maybe instead we can put "Consider the following `createUnicorn` function"?

Reply:
Good Catch, Travis! Thanks, updating now!